StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentMethodsView.swift

```diff
@@ -95,7 +102,7 @@ class EmbeddedPaymentMethodsView: UIView {
             let selection: Selection = .new(paymentMethodType: .stripe(.card))
             let cardRowButton = RowButton.makeForPaymentMethodType(
                 paymentMethodType: .stripe(.card),
-                savedPaymentMethodType: savedPaymentMethod?.type,
+                hasSavedCard: !savedPaymentMethods.filter{$0.type == .card}.isEmpty,
```

Comment:
To avoid duplicating this logic, we can add:
```
extension Array where Element == STPPaymentMethod {
    func hasSavedCard() -> Bool {
        return !self.filter{$0.type == .card}.isEmpty
    }
}
```

Reply:
cool! made it a var though